packages/@aws-cdk/custom-resources/lib/aws-custom-resource.ts

```diff
@@ -124,6 +124,13 @@ export interface AwsCustomResourceProps {
    * @default extract the permissions from the calls
    */
   readonly policyStatements?: iam.PolicyStatement[];
+
+  /**
+   * The timeout for the Lambda function implementing this custom resource.
+   *
+   * @default Duration.seconds(6)
```

Comment:
Is 6 seconds realistic? Should we do 30 as a default?

Reply:
OK for 30.